src/yoroi-wallets/storage.ts

```diff
@@ -40,12 +40,21 @@ const mountStorage = (path: Path) => {
     removeItem: (key: string) => {
       return AsyncStorage.removeItem(withPath(key))
     },
+    removeFolder: async (folderName: FolderName) => {
```

Comment:
so wallet manager can delete all of a wallets plaintext storage
no including encrypted storage, or auth with os storage